src/lake/Lake/Build/Library.lean

```diff
@@ -73,6 +73,20 @@ def LeanLib.staticFacetConfig : LibraryFacetConfig staticFacet :=
 def LeanLib.staticExportFacetConfig : LibraryFacetConfig staticExportFacet :=
   mkFacetJobConfig (LeanLib.recBuildStatic · true)
 
+@[specialize] protected def LeanLib.buildStaticFat
+(self : LeanLib) : FetchM (BuildJob FilePath) := do
+    withRegisterJob s!"{self.name}:static.fat" do
+    let mods ← self.rootModules.concatMapM fun mod => do
+      return (← mod.transImports.fetch).push mod
```

Comment:
To avoid aborting the entire build process on ill-formed imports, this be wrapped in a `try ... catch` block like the following example from `Module`:

https://github.com/leanprover/lean4/blob/fe0cb97c5d4d43393728ddd885415bc5bfb4b929/src/lake/Lake/Build/Module.lean#L111-L112